Core/Utilities/InstallTools.py

```diff
@@ -210,6 +211,8 @@ def loadDiracCfg( verbose = False ):
   if verbose and monitoringClient:
     gLogger.notice( 'Client configured for Component Monitoring' )
 
+  componentTypes = [ 'service', 'agent', 'executor' ]
```

Comment:
Maybe this could be put on top of the module and every place it's used, put the global one?